### ISSUE TITLE
[BUG] "Invalid private key" for ^1.20.0 when DOTENVX_PRO_USER set

### ISSUE THREAD

**Issue by gander** [dotenvx/dotenvx#446]
```shell
rm .env*
touch .env
dotenvx set HELLO World
```

Exits with `Invalid private key` message and `0` status for:

- 1.20.0
- 1.20.1
- 1.21.0
- 1.21.1
- 1.22.0
- 1.22.1
- 1.22.2
- 1.23.0

When `$HOME/.config/dotenvx/.env` contains `DOTENVX_PRO_USER`

**Comment by motdotla** [comment]
are you using dotenvx-pro @gander? it's still under development so I'd recommend just removing it for now and clearing out your ~/.config/dotenvx/.env file

**Comment by gander** [comment]
@motdotla yes, I'm testing pro, but as long as nothing interesting for me is happening there, I can delete it

**Comment by gander** [comment]
I changed the name to `.env~pro` to be able to test it in the future and so that it does not conflict with projects.

**Comment by gander** [comment]
But as part of pro development you can verify this case. Originally I was just going to report a problem with these versions, but I decided to check if it was a pro issue, and actually checked which value was causing problems. Use this research.

**Comment by motdotla** [comment]
ok thank you, i appreciate that. will investigate soon.

**Comment by motdotla** [comment]
if you download the latest pro and run `dotenvx pro login` i believe it should fix this problem up for you.

```
$ curl -fsS https://dotenvx.sh/pro | sh
$ dotenvx pro login
```
